packetbeat/protos/protos.go

```diff
@@ -231,11 +231,17 @@ func (protocols ProtocolsStruct) BpfFilter(with_vlans bool, with_icmp bool) stri
 
 func (protos ProtocolsStruct) Register(proto Protocol, plugin ProtocolPlugin) {
 	protos.all[proto] = plugin
+	var success bool = false
```

Comment:
@medcl You're right. Ignore my comment, I read that block as if it was an if/else which it is clearly not.